kratos/tests/test_array_1d_interface.py

```diff
@@ -3,7 +3,12 @@
 
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 
+import sys
+
 class TestArray1DInterface(KratosUnittest.TestCase):
+    def setUp(self):
+        if (sys.version_info < (3, 2)):
+            self.assertRaisesRegex = self.assertRaisesRegexp
```

Comment:
@maceligueta I just checked, I think if you add this to the `__init__` of `KratosUnittest.TestCase` it should be fixed globally for all tests.
Could you try please?